python/uagents-core/uagents_core/utils/registration.py

```diff
@@ -52,7 +52,7 @@ def _send_post_request(
         return True, response
     except requests.RequestException as e:
         logger.error(
-            msg="Error submitting request",
+            msg=f"Error submitting request: {response.text}",
```

Comment:
but we still have raise for status right? so if that triggers, then the response won't be defined no?